service/src/main/java/bio/terra/cbas/controllers/MethodsApiController.java

```diff
@@ -102,19 +102,19 @@ public ResponseEntity<PostMethodResponse> postMethod(PostMethodRequest postMetho
       if (rawMethodUrl == null || rawMethodUrl.isEmpty()) {
         String errorMsg =
             "Error while importing Dockstore workflow. No workflow url found at specified path.";
-        log.error(errorMsg);
+        log.warn(errorMsg);
         return new ResponseEntity<>(
             new PostMethodResponse().error(errorMsg), HttpStatus.BAD_REQUEST);
       }
     } catch (URISyntaxException | MalformedURLException e) {
       String errorMsg =
           "Bad user request. Method url has invalid value. Error: %s".formatted(e.getMessage());
-      log.error(errorMsg, e);
+      log.warn(errorMsg, e);
       return new ResponseEntity<>(new PostMethodResponse().error(errorMsg), HttpStatus.BAD_REQUEST);
     } catch (bio.terra.dockstore.client.ApiException e) {
       String errorMsg =
           "Error while importing Dockstore workflow. Error: %s".formatted(e.getMessage());
-      log.error(errorMsg, e);
+      log.warn(errorMsg, e);
```

Comment:
This kind of thing still feels like a metric to track it would be super helpful.